package.json

```diff
@@ -50,7 +50,8 @@
     "eslint": "^3.15.0",
     "eslint-config-airbnb-base": "^11.1.0",
     "eslint-plugin-import": "^2.2.0",
-    "jest": "^20.0.4"
+    "jest": "^20.0.4",
+    "jest-environment-node-debug": "^2.0.0"
```

Comment:
this is somewhat controversial - it's required to make the debugging of jest tests in the terminal work. However I'm perfectly happy just to install it everytime for myself - I'd welcome any thoughts on this :)